utils/movies.php

```diff
@@ -0,0 +1,53 @@
+<?php 
+require "database.php";
+// Upload movie function
+
+// example: uploadMovie(1, "title", "asdfg", "p" , 1, "asdfg", 1, 3);
+function uploadMovie($userId, $title, $path, $thumbnailPath, $description, $ageRating, $filmGuides, $genres){
+    $title = filterInputPost($_POST['MovieTitle'], 'MovieTitle');
+    $genre = filterInputPost($_POST['Category'], 'Category'); 
```

Comment:
lijn 8 en 18, je filtert 2x dezelfde variabele. Hetzelfde voor lijn 10 en lijn 23. Als je tegen een error aanloopt hiermee gaat er waarschijnlijk in de naamgeving iets niet helemaal lekker. 

gaat dit trouwens wel goed voor Genre (lijn 8) en filmguide (lijn 10) met het filteren?

Beide waardes komen binnen als een array omdat het meerdere waardes kan bevatten. Een film kan bijv 2 of meer genres hebben. De functie die je gebruikt om te filteren accepteert alleen een string, geen array